fedimint-api-client/src/api/mod.rs

```diff
@@ -237,6 +237,7 @@ pub trait FederationApiExt: IRawFederationApi {
         }
     }
 
+    #[instrument(target = LOG_CLIENT_NET_API, level = "debug", skip(self, strategy))]
```

Comment:
`params` here could potentially be quite large (?) maybe? Are they really useful or maybe should we skip?